### ISSUE TITLE
Screen reader is not announcing any success message upon hitting enter key on 'Refresh Target' or 'Reload' button.

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode#137788]
GitHubTags:#A11yWCAG2.1;#A11ySev3;#A11yTCS;#A11yMAS;#DesktopApp;#WCAG4.1.3;#MAC;#AINotInScope;#VoiceOver;#BM-VisualStudioCodeClient-MAC-Nov2021;#A11y_Visual Studio Code Client;#Linked:Bug28807826;
### Environment Details:​
Application Name: Visual Studio Code
Standard Violated: Status message

### Repro Steps:​
1.Open Visual Studio Code .
2.Tab till extensions and install "Microsoft Edge(Chromium) Tools".
3. Navigate to 'Microsoft Edge(Chromium) Tools' tab item and hit enter key.
4. Navigate to 'Open a New Tab' button and hit enter key.
5.  Enter web address for e.g;(Bing.Com) in address bar and hit enter key.
6. Now navigate back to visual studio code application using ALT+TAB key or minimize the browser.
7. Navigate to "Refresh Target" button in and hit enter key. 
8. Verify whether Screen reader is announcing the completion status message or not.

### Actual Result:
Voice Over is not announcing the refresh completion status after completing the refresh.

### Expected Result: ​
Voice Over should announce the refresh completion status like "Refresh completed"  or "Refreshed successfully" status message after completion of refresh.

### User Impact:
Screen reader users might face difficulty in understanding whether action performed successfully or not.
### ![Voice over is not announcing the  refreshed successfully  status message ](https://user-images.githubusercontent.com/89508906/143230039-ba16678e-44f1-4575-a8a6-d1c8d75c764d.gif)

**Comment by Latha2564** [comment]
Primary bug:https://microsoft.visualstudio.com/Edge/_workitems/edit/28807826

**Comment by isidorn** [comment]
Yes, actions in vscode do not report success status. This is just a design decision we did, and so far we have not heard any user request for this. Only request is for long running file operations and there is already an issue tracking this.
Thus closing as designed.
